File: eBotanika.Services/Database/Placanje.cs
namespace eBotanika.Services.Database
{
    public partial class Placanje
    {
        public int PlacanjeID { get; set; }
        public int KorisnikId { get; set; }
        public Korisnik Korisnik { get; set; }
        public string BrojTransakcije { get; set; }
        public string Currency { get; set; }
        public double Iznos { get; set; }
    }
}

File: ebotanika_mobile/lib/models/placanje.dart
class Placanje {
  int korisnikID;
  String brojTransakcije;
  String currency;
  double iznos;

  Placanje(
      {required this.korisnikID,
      required this.brojTransakcije,
      required this.currency,
      required this.iznos});

  factory Placanje.fromJson(Map<String, dynamic> json) {
    return Placanje(
        korisnikID: json["korisnikID"],
        brojTransakcije: json["brojTransakcije"],
        currency: json["currency"],
        iznos: json["iznos"]);
  }

  Map<String, dynamic> toJson() => {
        "korisnikID": korisnikID,
        "brojTransakcije": brojTransakcije,
        "currency": currency,
        "iznos": iznos
      };
}
